manifests/profile/www_lib/apache.pp

```diff
@@ -119,6 +119,8 @@
     }
   }
 
+  include nebula::profile::www_lib::vhosts::publishing
```

Comment:
Any reason this isn't part of the above vhost array?

Reply:
These things don't follow our pattern of flexing the prefix/domain at all, so I decided to not add unused and empty params to the new profile. The resources in there use `apache::vhost` directly, too.